src/test/java/IvanFFavoriteTests.java

```diff
@@ -0,0 +1,109 @@
+import PagesIvaF.AllSongsIvan;
+import PagesIvaF.AllSongsIvan;
+import PagesIvaF.HomePageIvan;
+import PagesIvaF.LoginPageIvan;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedCondition;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import org.testng.annotations.TestInstance;
+import java.time.Duration;
+
+
+public class IvanFFavoriteTests extends BaseTestIvan{
```

Comment:
Please, avoid own names or vague words: If there’s not a real need for them, avoid using these words, which often add no value

For exampe

`class FavoriteTests`